convert/gitlab/convert_test.go

```diff
@@ -15,24 +15,26 @@
 package gitlab
 
 import (
+	"fmt"
 	"io/ioutil"
 	"path/filepath"
+	"sort"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 	"gopkg.in/yaml.v3"
 )
 
 func TestConvert(t *testing.T) {
-	tests, err := filepath.Glob("testdata/*/*.yaml")
+	tests, err := filepath.Glob("yaml/testdata/*.yaml")
```

Comment:
the conversion package should have its own `testdata` folder so that the yaml golden files are the harness yaml.  it looks like the GitLab yaml parser and converter has poor / almost zero coverage, but we should follow the same patterns as the other packages.

Reply:
done.